docs/_sources/week02/spring-application/index.rst.txt

```diff
@@ -0,0 +1,239 @@
+    - Configure Project Dependencies in Gradle
+    - Intialize a New Git Repository
+    - Connect our local git repository to a remote repository on GitLab
+    - Setup PostGIS
+    - Create Databases, and Database users
+    - Configure Spring Data
+
+Create Spring Project
+---------------------
+
+We will be creating a Spring project from scratch today! Luckily for us Pivotal, the owners of Spring, have tried to make it as easy as possible to create a new Spring project. 
+
+In your browser navigate to `start.spring.io <https://start.spring.io>`_.
+
+From the Spring Initalizr we have some options to create a ready to go spring application!
+
+You will need to select:
+    - Project: *Gradle*
+    - Language: *Java*
+    - Spring Boot: *2.1.X*
+    - Project Metadata Group: *org.launchcode*
+    - Project Metadata Artifact: *zikaDashboard*
+    - Project Metadata Name: *zikaDashboard*
+    - Project Metadata Description: *CDC Zika Dashboard*
+    - Project Metadata Package Name: *org.launchcode.zikaDashboard*
+    - Project Metadata Packaging: *Jar*
+    - Project Metadata Java Version: *8*
+    - Dependencies: *Web*
+    - Dependencies: *Thymeleaf*
+    - Dependencies: *JPA*
+    - Dependencies: *DevTools*
+
+Double check that you have filled out the informatoin correctly, and then click Generate Project. This will prompt your browser to download a file called zikaDashboard.zip, go ahead and download it, and take notice of where it is downloaded. Most machines default to a Download directory off of your Home directory.
+
+We will want to move the zikaDashboard.zip file to wherever you typically store your projects, and then unzip the file.
+
+You command will likely be different, but I will move my zikaDashboard.zip file to my projects folder with the following command.
+
+.. sourcecode:: console
+   :caption: Move & unzip zikaDashboard.zip
+
+   $ mv /home/paul/Downloads/zikaDashboard.zip /home/paul/projects/
+   $ cd /home/paul/projects
+   $ unzip zikaDashboard.zip
+
+Upon unzipping the file, I will have a new folder called zikaDashboard. Looking into the folder you will see a familiar project structure for Spring projects.
+
+Open Project in IntelliJ
+------------------------
+
+Now that we have a directory with our Spring application in it, let's go ahead and open this project with IntelliJ.
+
+From IntelliJ Open ``build.gradle`` as a new project, make sure use auto-import is selected!
+
+After opening your project look around, it should look very familiar to the other projects we have worked on in this class.
+
+Configure Project Dependencies
+------------------------------
+
+In creating our Spring project we added four dependencies directly from Spring: Web, Thymeleaf, JPA, and DevTools.
+
+However, we have 2 additional dependencies we will need: a PSQL Driver, and Hibernate. These two tools are not managed by Spring, and cannot be loaded through the Spring Intilizr, so we will have to manually include their dependencies.
```

Comment:
Intilizr typo

Reply:
I caught an Initializr typo in 4 different places in this file... Spelled 2 or 3 different ways, all of which were incorrect. Spring Initializr is the correct spelling.